### PR TITLE
Hinge

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

continuous of this [PR](https://github.com/bioconda/bioconda-recipes/pull/3556)

### CODE REVIEW

**Comment by mictadlo** [comment]
@druvus Thank you for spdlog. I patched all files for spdlog, but it failed do build. Any idea what did I miss @bioconda/all ?

**Comment by bgruening** [comment]
@mictadlo please try to read the travis log. It seems this is the error: https://travis-ci.org/bioconda/bioconda-recipes/jobs/196433602#L655

File: recipes/hinge/build.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export BOOST_INCLUDE_DIR=${PREFIX}/include
+export BOOST_LIBRARY_DIR=${PREFIX}/lib
+
+#export CXXFLAGS=" -std=c++11 -stdlib=libstdc++ -stdlib=libc++ -DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export CXXFLAGS=" -std=c++11  -DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export LDFLAGS="-L${BOOST_LIBRARY_DIR}"
+export Boost_NO_BOOST_CMAKE=TRUE)
```
Comment (bgruening) [inline recipes/hinge/build.sh]: And the error brings you to this line ...

Comment (bgruening) [inline recipes/hinge/build.sh]: Also remember if you ping bioconda/all you will ping nearly 200 people, please use this only in cases where you really want to ping all!

Comment (mictadlo) [inline recipes/hinge/build.sh]: Thank you for the tip and sorry to  ping all.

File: recipes/hinge/build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export BOOST_INCLUDE_DIR=${PREFIX}/include
+export BOOST_LIBRARY_DIR=${PREFIX}/lib
+
+#export CXXFLAGS=" -std=c++11 -stdlib=libstdc++ -stdlib=libc++ -DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export CXXFLAGS=" -std=c++11  -DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export LDFLAGS="-L${BOOST_LIBRARY_DIR}"
+export Boost_NO_BOOST_CMAKE=TRUE 
+
```
Comment (bgruening) [inline recipes/hinge/build.sh]: Maybe this helps: https://travis-ci.org/bioconda/bioconda-recipes/jobs/196444935#L706
`BOOST_LIBRARYDIR`

Comment (mictadlo) [inline recipes/hinge/build.sh]: Thank you

**Comment by fxia22** [comment]
see https://github.com/fxia22/HINGE/issues/90

**Comment by mictadlo** [comment]
@bgruening is `/opt/conda/conda-bld/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/include` the correct path to BOOST?

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**